main.py

```
@@ -326,135 +581,674 @@ def run_plugin(self):
+    # Инициализация сбора точек
+    def start_point_collection():
+        
+        collector = CustomDEMPlugin.PointCollector(iface.mapCanvas())
+        iface.mapCanvas().setMapTool(collector)
+        print("Щелкните по карте, чтобы выбрать точки.")
+        
+        return collector
+    '''
+    ======================= CREATING FOREST BELT===========================
+    '''
+    def add_dem_layer(output_path):
```

Ruff (0.8.2)

Redefinition of unused `add_dem_layer` from line 158

(F811)